packages/@react-aria/interactions/src/useHover.ts

```diff
@@ -16,12 +16,11 @@
 // See https://github.com/facebook/react/tree/cc7c1aece46a6b69b41958d731e0fd27c94bfc6c/packages/react-interactions
 
 import {HoverEvents} from '@react-types/shared';
-import {HTMLAttributes, useMemo, useRef, useState} from 'react';
+import {HTMLAttributes, useEffect, useMemo, useRef, useState} from 'react';
 
 export interface HoverProps extends HoverEvents {
   /** Whether the hover events should be disabled. */
-  isDisabled?: boolean,
-  isHovered?: boolean
```

Comment:
Removed this since the prop didn't seem to be used anywhere. Was added in #775. We can add back later if needed.